CIP-0XXX/README.md

```diff
@@ -0,0 +1,561 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
+Status: Proposed
+Authors:
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - ?
+Implementors: 
+    - ?
```

Comment:
Could name the abstract "Cardano scaling" group if we want:

https://github.com/cardano-foundation/CIPs/pull/870/files#diff-4877302f625cfbd6c50c9948ca0dda37abcaf3352ffbd68e59b58256095b6395R11

Otherwise, just leave it empty with 

```suggestion
Implementors: []
```

Reply:
I mentioned the Cardano Scaling team :+1: